PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -101,20 +101,8 @@ public ApiClient(HttpClient client = null)
 
             HttpResponseMessage response = await Client.PostAsync(
                 $"{Config.Host}/api/{type}", content);
-            
-            if (response.IsSuccessStatusCode)
-            {
-                string data = await response.Content.ReadAsStringAsync();
-                List<T> result = ParseResponse<T>(data, type);
-                return result[0];
-            } else
-            {
-                string error = response.Content.ReadAsStringAsync().Result;
-                Console.WriteLine(
-                    $"Error: {error}"
-                );
-            }
-            return default(T);
+
+            return response;
```

Comment:
So, if I'm reading this correctly, we now return the response regardless of the status code and handle errors at a higher (closer to the UI) layer. Is that accurate?

Reply:
Yes. The touch table itself needs to know if a classification failed. If a classification fails to be created, the classification is put into a queue on the table and is resubmitted later. It happens in a catch block here -> https://github.com/zooniverse/galaxy-zoo-touch-table/pull/62/files#diff-db299c9cb11045d226153a1a8345ce8dR42